config/power_virtual_servers_regions.yml

```
@@ -0,0 +1,40 @@
+us-south:
+  :name: us-south
+  :description: Dallas, TX United States (deprecated)
```

Let's remove the "(deprecated)" note here, since it's not labeled as deprecated anywhere in the docs or web UI.
```suggestion
  :description: Dallas, TX United States
```